tests/test_relax_path_checks.py

```diff
@@ -25,10 +25,12 @@ class ToolArgparse(unittest.TestCase):
 '''
 
     def test_spaces_in_input_files(self):
-        with NamedTemporaryFile(mode='w') as f:
+        with NamedTemporaryFile(mode='w', delete=False) as f:
```

Comment:
Actually we are opening this file to write and later in the execution reading it  without closing. On linux its possible but on windows it gave errors so I did this.